### DISCUSSION TITLE
Merge without text in quotes

### DISCUSSION THREAD

**Discussion by jasonhudgens20** [WinMerge/winmerge#972] [Q&A]
Hello
I have two files. I would like to merge ONLY the texts that are NOT in quotes (" ") from two files.
What is the best way to do this?

Example:

I am a "good" guy
I am not a "bad" guy

Now I merge the first line (text) into the second and the result should be:
=> I am a "bad" guy
The word in " " was ignored.

**Comment by sdottaka** [comment]
WinMerge doesn't have that features.
However, it has the ability to merge selected in-line differences. You will get the results you want by doing the following:

![972](https://user-images.githubusercontent.com/98126/133170322-a4d3db20-a2d4-41a3-8144-c516893cc4ba.gif)
